airflow/cli/cli_parser.py

```diff
@@ -626,6 +626,20 @@ def positive_int(value):
     ),
     action="store_true",
     default=False)
+ARG_CONN_IMPORT = Arg(
+    ('file',),
+    help='Import connections from a file. Acceptable file formats .json, .yaml, .env',
+    type=str)
```

Comment:
```suggestion
    type=str)
```
Have you considered using argparse.Filetype?
https://docs.python.org/3/library/argparse.html#argparse.FileType
This way, you will be able to export data directly to stdout.
```
airflow connections export - | grep MY_CON
```Have you considered using argparse.Filetype?
https://docs.python.org/3/library/argparse.html#argparse.FileType
This way, you will be able to import data directly from stdout.
```
cat my-file.json | airflow connections export - --format=json
```

Reply:
Added this in the PR